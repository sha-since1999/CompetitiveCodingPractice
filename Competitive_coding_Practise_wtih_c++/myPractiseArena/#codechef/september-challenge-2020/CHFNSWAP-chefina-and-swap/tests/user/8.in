10
732831472
577810315
482798088
653968604
131142432
385673794
323974331
589014247
6050490
5426798