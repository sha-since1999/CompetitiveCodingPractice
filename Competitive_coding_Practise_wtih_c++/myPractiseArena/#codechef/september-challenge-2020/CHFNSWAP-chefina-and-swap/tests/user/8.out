214641369
169236724
141408287
191542970
38410730
0
94889885
172518279
0
0
